axelor-redmine/src/main/resources/views/TimesheetLine.xml

```diff
@@ -13,7 +13,111 @@
 				<field name="durationUnit" hidden="true" />
 			</insert>
 		</extend>
+		
+		<extend target="//field[@name='duration']">
+			<attribute name="hidden" value="true"/>
+			<move position="before" source="//field[@name='hoursDuration']"/>
+		</extend>
+		
+		<extend target="//field[@name='hoursDuration']">
+			<attribute name="onChange" value="action-timesheet-line-method-set-duration"/>
+			<attribute name="hidden" value="false"/>
+		</extend>
+	</grid>
+	
+	<grid id="redmine-timesheet-line-timesheet-manuf-order-grid" name="timesheet-line-timesheet-manuf-order-grid" title="Timesheet Lines" 
+	model="com.axelor.apps.hr.db.TimesheetLine" extension="true">
+		<extend target="//field[@name='duration']">
+			<attribute name="hidden" value="true"/>
+			<move position="before" source="//field[@name='hoursDuration']"/>
+		</extend>
+		
+		<extend target="//field[@name='hoursDuration']">
+			<attribute name="onChange" value="action-timesheet-line-method-set-duration"/>
+			<attribute name="hidden" value="false"/>
+		</extend>
+	</grid>
+	
+	<grid id="redmine-timesheet-line-timesheet-operation-order-grid" name="timesheet-line-timesheet-operation-order-grid" title="Timesheet Lines" 
+	model="com.axelor.apps.hr.db.TimesheetLine" extension="true">
+		<extend target="//field[@name='duration']">
+			<attribute name="hidden" value="true"/>
+			<move position="before" source="//field[@name='hoursDuration']"/>
+		</extend>
+		
+		<extend target="//field[@name='hoursDuration']">
+			<attribute name="hidden" value="false"/>
+			<attribute name="onChange" value="action-timesheet-line-method-set-duration"/>
+		</extend>
 	</grid>
+	
+	<grid id="redmine-timesheet-line-timesheet-readonly-manuf-order-grid" name="timesheet-line-timesheet-readonly-manuf-order-grid" title="Timesheet Lines" 
+	model="com.axelor.apps.hr.db.TimesheetLine" extension="true">
+		<extend target="//field[@name='duration']">
+			<attribute name="hidden" value="true"/>
+			<move position="before" source="//field[@name='hoursDuration']"/>
+		</extend>
+		
+		<extend target="//field[@name='hoursDuration']">
+			<attribute name="hidden" value="false"/>
+		</extend>
+	</grid>
+	
+	<grid id="redmine-timesheet-line-daily-timesheet-grid" name="timesheet-line-daily-timesheet-grid" title="Daily timesheet lines"
```

Comment:
Make direct change in timesheet-line-daily-timesheet-grid at axelor-dailyts module